### DISCUSSION TITLE
Multilable segmentation with DiceMetric

### DISCUSSION THREAD

**Discussion by greenyjwk** [Project-MONAI/MONAI#7820] [Q&A]
Hello,

I want to find each class dice score using DiceMetric instead of finding whole Dice score.

My prediction dimension is (1,4,255,255,255 ) == (Batch, Class, Width, Height, Depth)
and
My label dimension is (1,1,255,255,255 ) == (Batch, , Width, Height, Depth) 
Background:0, Kidney:1, Tumor:2, Cyst:3

Using DiceMetric(monai.metrics.DiceMetric) I want to find the dice score for **only Tumor** dice score. Which arguments would I need to change from DiceScore for this task?

**Comment by KumoLiu** [comment]
Hi @greenyjwk, you can set `reduction="mean_channel"` to get dice for each channel.
https://github.com/Project-MONAI/MONAI/blob/4029c422f1da84cdf5e4ce546dbff33245cbe025/monai/metrics/meandice.py#L42

Hope it helps, thanks.
